extension/cart/fetchShopifyCheckout.js

```diff
@@ -32,14 +33,17 @@ module.exports = async (context, input) => {
 async function fetchCheckout (shopifyApiRequest, createNew, context) {
   const checkoutToken = await loadCheckoutToken(context)
   let checkout
+  let isNew
   try {
     if (!createNew && checkoutToken) {
       checkout = await shopifyApiRequest.getCheckout(checkoutToken)
+      isNew = false
     } else {
       checkout = await shopifyApiRequest.createCheckout()
+      isNew = true
     }
 
-    return ({ isNew: createNew, checkout })
+    return ({ isNew, checkout })
```

Comment:
why are there `( )` surrounding this return, what does it do?

Reply:
Should do nothing, removed it.